docs/cookbook/descriptors.md

```diff
@@ -2,7 +2,7 @@
 BDK is a descriptor-first library. This page explores how to build them and how they interact with other standards like BIP-39 recovery phrases.
 
 !!!danger
-    TODO: Standard notice of danger of working with private keys, generating entropy, etc.
+    The keys and descriptors used in **The Book of BDK** are for illustration purposes only; **UNDER NO CIRCUMSTANCES** should any of the keys or descriptors containing private data be used for real money. Best practice for generating entropy is to do so in a controlled environment where information won't be exposed to untrusted networks.
```

Comment:
Also mention that you should rely on a [cryptographically secure pseudorandom number generator (CSPRNG)](https://en.wikipedia.org/wiki/Cryptographically_secure_pseudorandom_number_generator)

Note that all the milksad.info exposed users where following the directives above, but Libbitcoin Explorer (`bx`) used a non-CS PRNG and with only 32 bits of entropy based on system time.

Reply:
That's a great point